runtime/doc/cmdline.txt

```diff
@@ -919,7 +919,7 @@ option contains "sh", this is done twice, to avoid the shell trying to expand
 the "!".
 
 							*filename-backslash*
-For filesystems that use a backslash as directory separator (MS-DOS and
+For filesystems that use a backslash as directory separator (NTFS on
```

Comment:
Well I guess FAT right? Maybe `Windows filesystems`? There are a couple of places to update for this wording BTW.